libsolidity/parsing/Parser.cpp

```diff
@@ -94,14 +94,22 @@ ASTPointer<SourceUnit> Parser::parse(CharStream& _charStream)
 		m_recursionDepth = 0;
 		m_scanner = make_shared<Scanner>(_charStream);
 		ASTNodeFactory nodeFactory(*this);
+		bool finishedParsingPragmas = false;
+		m_experimentalSolidityEnabledInCurrentSourceUnit = false;
```

Comment:
Why not just the below? Does we really need to be explicit about it been enabled in the current source unit? I mean, if it is enable, it should be for all sources, right?

```suggestion
		m_experimentalSolidityEnabled = false;
```